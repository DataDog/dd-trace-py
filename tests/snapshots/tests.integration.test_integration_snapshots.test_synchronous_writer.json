[[
  {
    "name": "operation2",
    "service": "my-svc",
    "resource": "operation2",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "meta": {
      "runtime-id": "7fcf456c657a442d8e637e5adda7270d"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 52166
    },
    "duration": 81000,
    "start": 1631827407420781000
  },
     {
       "name": "child2",
       "service": "my-svc",
       "resource": "child2",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "duration": 12000,
       "start": 1631827407420838000
     }],
[
  {
    "name": "operation1",
    "service": "my-svc",
    "resource": "operation1",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "meta": {
      "runtime-id": "7fcf456c657a442d8e637e5adda7270d"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 52166
    },
    "duration": 77000,
    "start": 1631827407415915000
  },
     {
       "name": "child1",
       "service": "my-svc",
       "resource": "child1",
       "trace_id": 1,
       "span_id": 2,
       "parent_id": 1,
       "duration": 13000,
       "start": 1631827407415968000
     }]]
