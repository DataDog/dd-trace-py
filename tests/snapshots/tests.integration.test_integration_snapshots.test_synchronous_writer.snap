[[{"name" "operation1"
   "service" "my-svc"
   "resource" "operation1"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1616457150270038000
   "duration" 47000
   "meta" {"runtime-id" "ddf562510b004d60bdfaa21911ab1e60"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 39631
              "_dd.tracer_kr" 1.0}}
      {"name" "child1"
       "service" "my-svc"
       "resource" "child1"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1616457150270074000
       "duration" 7000}]
 [{"name" "operation2"
   "service" "my-svc"
   "resource" "operation2"
   "error" 0
   "span_id" 0
   "trace_id" 1
   "parent_id" nil
   "start" 1616457150273685000
   "duration" 46000
   "meta" {"runtime-id" "ddf562510b004d60bdfaa21911ab1e60"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 39631
              "_dd.tracer_kr" 1.0}}
      {"name" "child2"
       "service" "my-svc"
       "resource" "child2"
       "error" 0
       "span_id" 1
       "trace_id" 1
       "parent_id" 0
       "start" 1616457150273719000
       "duration" 8000}]]
